prometheus-client/src/Prometheus/Metric/Summary.hs

```diff
@@ -72,15 +75,15 @@ collectSummary info valueTVar = STM.atomically $ do
     estimator@(Estimator count itemSum _ _) <- STM.readTVar valueTVar
     let quantiles = map fst $ estQuantiles estimator
     let samples =  map (toSample estimator) quantiles
-    let sumSample = Sample (metricName info ++ "_sum") [] (bsShow itemSum)
-    let countSample = Sample (metricName info ++ "_count") [] (bsShow count)
+    let sumSample = Sample (metricName info <> "_sum") [] (bsShow itemSum)
+    let countSample = Sample (metricName info <> "_count") [] (bsShow count)
```

Comment:
You'll need to rebase / merge in recent master changes, as we've fixed a bug in this code and that fixes conflicts with these changes.